src/Codor/Sniffs/Files/ReturnNullSniff.php

```diff
@@ -35,10 +35,16 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
         });
 
         $returnValueIndex = key($semicolons) - 1;
+        $comparaisonIndex = key($semicolons) - 3;
```

Comment:
Could replace `comparaison` by `comparison` and rename the files accordingly please?